netmaster/master/netmaster_test.go

```diff
@@ -296,3 +308,33 @@ func TestVxlanConfigWithLateHostBindings(t *testing.T) {
 
 	verifyKeys(t, keys)
 }
+
+// Tests for https://github.com/contiv/netplugin/issues/214
+func TestVxlanConfigPktTagOutOfRange(t *testing.T) {
+	cfgBytes := []byte(`{
+    "Tenants" : [{
+        "Name"                  : "tenant-one",
+        "DefaultNetType"        : "vxlan",
+        "SubnetPool"            : "11.1.0.0/16",
+        "AllocSubnetLen"        : 24,
+        "Vxlans"                : "2001-3000",
+        "Networks"  : [{
+            "Name"              : "orange",
+            "PktTag"            : 1300
+        }]
+    }]}`)
+
+	initFakeStateDriver(t)
+	defer deinitFakeStateDriver()
+
+	// Create the Tenant and Network
+	applyConfig(t, cfgBytes)
```

Comment:
@vijayvikrant - got it... it seems like you plan to modify this test case once you fix issue; up to you if you want to combine this with other PR or keep them separate.
otherwise LGTM